src/shared/navgen/nav.cpp

```diff
@@ -415,8 +440,10 @@ void NavmeshGenerator::LoadTris( std::vector<float> &verts, std::vector<int> &tr
 	for ( int k = model->firstSurface, n = 0; n < model->numSurfaces; k++, n++ )
 	{
 		const dsurface_t *surface = &bspSurfaces[ k ];
+		const dshader_t *shader = &bspShaders[surface->shaderNum];
 
-		if ( !( bspShaders[surface->shaderNum].contentFlags & solidFlags ) ) {
```

Comment:
Why is the content flags check removed here?

Reply:
I've mistakenly wrote the content check in the wrong block, now moved back to its expected place.